xml/admin_gui_dashboard.xml

```diff
@@ -2327,6 +2413,130 @@ Oct 04 07:48:53 doc-ses6min4 systemd[1]: Started Ceph rbd mirror daemon.
 </screen>
     </tip>
    </sect3>
+   <sect3 xml:id="cert-sign-custom-CA">
+     <title>Certificates Signed with a Custom CA</title>
+     <para>
+       The following procedure needs to be followed once to create the root CA.
+     </para>
+     <note>
+        <para>
+          This is the key used to sign the certificate requests. Anyone
+           holding this can sign certificates on your behalf.
+         </para>
+      </note>
+     <procedure>
+       <step>
+         <para>
+           Create the Root Key:
+         </para>
+<screen>
+&prompt.cephuser;openssl genrsa -des3 -out rootCA.key 4096
+</screen>
+        <note>
+          <para>
+            If you want a non-password protected key, remove the
+            <option>-des3</option> option.
+          </para>
+        </note>
+       </step>
+       <step>
+         <para>
+           Create and self-sign the root certificate:
+         </para>
+<screen>
+&prompt.cephuser;openssl req -x509 -new -nodes -key rootCA.key -sha256 -days 1024 -out rootCA.crt
+</screen>
+       </step>
+     </procedure>
+     <para>
+       The following procedure needs to be followed for each server
+       that needs a trusted certificate from our CA.
+     </para>
+     <procedure>
+       <step>
+         <para>
+          Create the certificate key:
+        </para>
+<screen>
+&prompt.cephuser;openssl genrsa -out mydomain.com.key 2048
+</screen>
+	<para>
+           The certificate signing request is where you specify the details for
+           the certificate you want to generate. This request is processed by
+           the owner of the Root Key to generate the certificate.
+         </para>
+       </step>
+       <step>
+         <important>
+           <para>
+             When creating the certificate signing request, it is important to specify
+             the <literal>Common Name</literal> providing the IP address or
+             domain name for the service, otherwise the certificate cannot
+             be verified.
+           </para>
+         </important>
+         <para>
+           These are two ways to create the CSR:
+         </para>
+         <itemizedlist>
+           <listitem>
+             <para>
+               Interactive method:
+             </para>
+<screen>
+&prompt.cephuser;openssl req -new -key mydomain.com.key -out mydomain.com.csr
```

Comment:
I haven't tried this so I have no idea what to expect if I choose the interactive method. I'm guessing I will have to provide some additional information - what will that be?

Reply:
Basically you'll get a series of prompts. Either you go to the "interactive method" which prompts you to add in information (country name, locality name, etc)
If you go the one liner, you just put this information into the command. I've tried to clear that up.